server/endpoints/workspaces.js

```diff
@@ -321,6 +321,26 @@ function workspaceEndpoints(app) {
     }
   );
 
+  app.post(
+    "/workspace/chat-feedback",
+    [validatedRequest],
+    async (request, response) => {
+      try {
+        const { chatId, feedback } = reqBody(request);
+        const result = await WorkspaceChats.updateFeedbackScore(
+          Number(chatId),
+          Number(feedback)
```

Comment:
Do the casting inside of the method here, would prevent accidental non-cast if we use it later.